aries-site/src/examples/templates/wizard/WizardValidationExample.js

```diff
@@ -57,7 +59,7 @@ const validation = [
 
 const stepOneInputs = [
   (attemptedAdvance, error, formValues, setError) => (
-    <Box>
+    <Box width={{ max: 'medium' }}>
```

Comment:
These Box wrappers (L62 & L112) seem to be doing the same thing. Can we move inside the map at L144?

Reply:
Yes, good cleanup